### PR TITLE
turned down default command-line kl_ratio

### LINKED ISSUE #10: Parameter set for training over Hubble dataset
Saw your Deep Style presentation and the generated images based on trained model on Hubble dataset. I was wondering what parameter set you used to train over Hubble dataset. I used the default parameter set and only got black images generated at the end! Also played with image_multiplier argument and got a bunch of similar black images with a gray (lighter) circle in the center. 
Any tips here?

### PR DESCRIPTION
**Pull request by dribnet**
Dropped the default kl_ratio from 1.0 to 0.005. Note that
this change was only made to the command line default, not
to the VAE class initializer. With this change, the downloaded
hubble dataset produces reasonable images that don't all
look the same when fauxtograph train is run with 
default parameters.

Fixes #10 

Sample run below

```bash
fauxtograph train --gpu ./images ./models/hubble
fauxtograph generate ./models/hubble_model.h5 ./models/hubble_opt.h5 ./models/hubble_meta.json ./images_hubble
```

### CODE REVIEW

**Comment by tjtorres** [comment]
Hmmmm, I agree that with the default downloaded set I gave, performance is better with a bit lower number for the kl_ratio. The problem is that for more general purpose things with a larger image set you don't want to actually have it be too low to avoid overfitting. Maybe we can compromise at 0.1? If that doesn't work for the default set (though I believe I tested it and it seemed to work ok), then I'd be happy turning down to your value.

**Comment by dribnet** [comment]
Makes sense - based on this, I think simply editing the README instead makes more sense. I'll update this pull request to do that later.

Speaking of overfitting - what do you think of adding a validation set and reporting the NLL on the validation set during training? I've found that helps determine when to stop training and having validation NLL would also allow comparisons between papers or other implementations for the same dataset.

**Comment by dribnet** [comment]
rebased commit to just update README with lower `--kl_ratio` arg for the example train command.

New sample run (achieving same effect with `--kl_ratio` argument):
```bash
fauxtograph train --gpu --kl_ratio 0.005 ./images ./models/hubble2
fauxtograph generate ./models/hubble2_model.h5 ./models/hubble2_opt.h5 ./models/hubble2_meta.json ./images_hubble2
```

**Comment by tjtorres** [comment]
This seems like a reasonable solution. I'll merge it. As far the validation loss, I think it's a great idea. My thinking when I was building the command line tool was that it would mostly be for enthusiasts to generate some images quickly, but I am not averse to adding in some more advanced functionality if people wish to use it more seriously.
